### PR TITLE
Endless settings on templates and instances

### PR DESCRIPTION
**Pull request by Spaceman1984**
## Description

When loading the vm template settings from the ui, if there are more than twelve items in the list, the settings are loaded again if you scroll to the last setting.

This fixes the reload of the settings when scrolling to the end of the list.

Fixes: #3544 
## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## How Has This Been Tested?

This was tested by going to the settings tab of an instance and a template and adding many settings then navigating away and back and then scrolling down.

### CODE REVIEW

File: ui/scripts/instances.js [hunk 1]
```diff
@@ -3437,6 +3437,13 @@
 						custom: cloudStack.uiCustom.granularDetails({
                         resourceType: 'UserVm',
 							dataProvider: function(args) {
+                                // no paging for listVirtualMachines details
+                                if (args.page > 1){
+                                    args.response.success({
+                                        data: []
+                                    });
+                                    return;
+                                }
```
Comment (anuragaw) [inline ui/scripts/instances.js]: Can you fix indentation @Spaceman1984 ?

File: ui/scripts/templates.js [hunk 2]
```diff
@@ -2489,6 +2489,13 @@
 								custom: cloudStack.uiCustom.granularDetails({
                                     resourceType: 'Template',
 									dataProvider: function(args) {
+                                        // no paging for listTemplates details
+                                        if (args.page > 1){
+                                            args.response.success({
+                                                data: []
+                                            });
+                                            return;
+                                        }
```
Comment (anuragaw) [inline ui/scripts/templates.js]: same here.

File: ui/scripts/instances.js (hunk 1 above)
Comment (DaanHoogland) [reply ui/scripts/instances.js]: What do you mean @anuragaw ? I see no problem with the indentation here.

File: ui/scripts/templates.js (hunk 2 above)
Comment (DaanHoogland) [reply ui/scripts/templates.js]: or here?!

File: ui/scripts/instances.js (hunk 1 above)
Comment (anuragaw) [reply ui/scripts/instances.js]: Isn't this supposed to be inside dataProvider @DaanHoogland ?
image

Comment (Spaceman1984) [reply ui/scripts/instances.js]: The indentation is fine when viewing with vscode and intellij, I think this is a github display issue.

**Comment by DaanHoogland** [comment]
@borisstoyanov if you agree i'll merge

File: ui/scripts/instances.js (hunk 1 above)
Comment (anuragaw) [reply ui/scripts/instances.js]: What you see in IDEs is actually rendered as per configuration of whitespace. However, this is old UI and I've seen this broken at many files.

**Comment by anuragaw** [comment]
I've fixed indentation on the PR and LGTM

**Comment by DaanHoogland** [comment]
ui issue and 2 approvals, will merge.

**Pull request merged**